glass/src/lib/native/cpp/support/ExtraGuiWidgets.cpp

```diff
@@ -217,4 +222,68 @@ bool HamburgerButton(const ImGuiID id, const ImVec2 position) {
   return pressed;
 }
 
+struct InputExprState {
+  char* inputBuffer = nullptr;
+  bool wasActive = false;
+};
+
+static const int kBufferSize = 256;
+
+static std::unordered_map<int, InputExprState> exprStates;
+static char* previewBuffer =
+    reinterpret_cast<char*>(std::malloc(kBufferSize * sizeof(char)));
```

Comment:
This would likely be better as just `static char previewBuffer[kBufferSize];`.  It would be put in .bss and not force a function call at load or show up as a memory leak.

Reply:
Fixed!